src/graphs.rs

```diff
@@ -76,6 +76,13 @@ struct LockfileNpmGraphPackage {
   root_ids: HashSet<LockfilePkgId>,
   integrity: Option<String>,
   dependencies: BTreeMap<StackString, LockfileNpmPackageId>,
+  optional_dependencies: BTreeMap<StackString, LockfileNpmPackageId>,
```

Comment:
I think we’ll need to store optional peer dependencies now too (after my recent deno_npm pr)

Reply:
We could actually just store unresolved optional peer deps, but that might be risky in case we need it in the future.